src/result_processor.h

```diff
@@ -212,25 +226,37 @@ void RP_DumpChain(const ResultProcessor *rp);
  *            Timeout API
  ****************************************/
 
-#define rs_timercmp(a, b, CMP) \
-  (((a)->tv_sec == (b)->tv_sec) ? ((a)->tv_nsec CMP(b)->tv_nsec) : ((a)->tv_sec CMP(b)->tv_sec))
+static inline int rs_timercmp(struct timespec *a, struct timespec *b) {
```

Comment:
Now it is actually returning whether a is equal to or greater than b, right?
Do you think it would be better to rename it?